src/scripts/store.js

```diff
@@ -321,9 +72,43 @@ function convertToArray(obj) {
   });
 }
 
-export function getStoredList() {
-  const storedListLS = localStorage.getItem("destinations");
+//check/uncheck an item from a list
+export async function toggleButtonVisited(item) {
+  const visited = item.visited;
+  item.visited = !visited;
+
+  const query = new Parse.Query(Destinations);
+  query.equalTo("city", item.city);
```

Comment:
Here it's important to retrieve the destination by a unique id. If we create two items with the same city, this code would break because it's selecting the first city found. The same happens when deleting a destination

Reply:
That makes sense but I didn't find a way to do it, I thought I'd work around it with a city validation. ```equalTo()``` requires a key - value pair and at this point ```visited``` is the only thing I know the value of. I don't know the parse-created ```objectId``` and I didn't store the ids that were in the list because I didn't think I'd need them.  What would you do?

Reply:
@sandrina-p this is the one :)